### ISSUE TITLE
[BUG] Enderchests storing

### ISSUE THREAD

**Issue by SladeHazard** [snowgears/shopbugs#372]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Data loss, Other unexpected behaviour

### Environment

```markdown
- Server Version: 1.12.2 Paper
- Shop Version: 1.8.4-legacy
```

### Server startup log

N/A

### Shop config files

N/A

### Error log (if applicable)

N/A

### Bug description

Why is this plugin storing people's enderchest in a data folder? when it's supposed to be managed by world/playerdata folder files. is there any way I can disable this?

### Steps to reproduce

load this plugin on the server

### Expected behaviour

not store enderchests, it's unnecessary

### Actual behaviour

stores people's enderchests even if i clear their playerdata from the world folder

### Other information

also another I noticed since I have this plugin people have been getting each other's enderchest instead of their original ones after server restarts.

**Comment by snowgears** [comment]
You can turn off enderchests by removing ENDER_CHEST from the materials you can create shops from. I save enderchests because there are no native methods to access enderchest inventories of offline players with shops.

As for the "other information" section you provided, are you using another plugin like CMI to view offline enderchest inventories of other players by any chance?

**Comment by snowgears** [comment]
Fixed in 1.8.4.2 (in progress)

I am no longer saving ender chest inventories to a file and have switched to a new system where I load player DAT files and interact directly with NBT for offline enderchests.
